content/blog/2017/07/2017-07-03-contributor-summit.adoc

```diff
@@ -3,7 +3,7 @@ layout: post
 title: "Jenkins World Contributor Summit"
 tags:
 - event
-- jenkinsworld
+- jenkinsworld2017
```

Comment:
I think it makes sense to use both dated and non-dated tags in these posts. That way one could see all Jenkins World related posts ever, or just this year's